tests/cypress/e2e/actions_tasks3/case_105_cloud_storage.js

```diff
@@ -40,13 +40,26 @@ context('Cloud storage.', () => {
                     cy.get($el).should('exist');
                 });
             });
+        });
+
+        it('Check "Cloud Storage" manifest field.', () => {
             // Check add/remove manifest file
             cy.get('.cvat-add-manifest-button').should('be.visible').click();
             cy.get('[placeholder="manifest.jsonl"]').should('exist').should('have.attr', 'value', '');
             cy.get('[placeholder="manifest.jsonl"]').type(dummyData.manifest);
             cy.get('[placeholder="manifest.jsonl"]').should('have.attr', 'value', dummyData.manifest);
             cy.get('[data-icon="delete"]').should('be.visible').click();
             cy.get('[placeholder="manifest.jsonl"]').should('not.exist');
+
+            // Check we can't add non-jsonl file
+            cy.get('.cvat-add-manifest-button').should('be.visible').click();
+            cy.get('[placeholder="manifest.jsonl"]').type('manifest.json');
```

Comment:
I do not think it is a good idea to select manifest field by placeholder attribute